recipe/meta.yaml

```diff
@@ -54,35 +53,134 @@ requirements:
+    requirements:
+      build:
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+      host:
+        - python
+        - gmp
+        - pybind11
+        - jsoncpp
+      run:
+        - python
+        - sympy
+        - matplotlib-base
+    files:
+      - lib/python*/site-packages/cdb/**/*
+      - lib/python*/site-packages/cadabra2.so
+      - lib/python*/site-packages/cdb_appdirs.py
+      - lib/python*/site-packages/cadabra2_defaults.py
+      - share/man/man1/cadabra2.1
+      - share/cadabra2/**/*
+      - bin/cadabra2
+    test:
+      imports:
+        - cadabra2
+      commands:
+        - test -f $PREFIX/share/cadabra2/python/cadabra2.so
+        - test -f $PREFIX/share/cadabra2/latex/preamble.tex
+
+  - name: cadabra2-gtk
+    requirements:
+      build:
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+      host:
+        - gtkmm-3.0
+        - glibmm
+        - cairomm-1.0
+        - glib
+        - sigcpp-2.0
+        - pangomm-1.4
+        - cairo
+        - atkmm-1.6
+        - jsoncpp
+        - boost-cpp
+        - sqlite
+        - python
+      run:
+        # Remove these when they get run_exports
+        - {{ pin_compatible("cairo") }}
+        - {{ pin_compatible("glibmm") }}
+        - {{ pin_compatible("glib") }}
+        - boost-cpp
+        - python
+        - {{ pin_subpackage("cadabra2", exact=True) }}
+      run_constrained:
+        - cadabra-jupyter-kernel {{ version }}
+    files:
+      - bin/cadabra2-gtk
+      - bin/cadabra2latex
+      - bin/cadabra-server
+      - bin/cadabra2cadabra
+      - bin/cadabra2html
+      - bin/cadabra2python
+      - bin/cadabra2-cli
+      - share/icons/**/cadabra2*
+      - share/man/man1/cadabra2-gtk.1
+      - share/man/man1/cadabra-server.1
+      - share/man/man1/cadabra2cadabra.1
+      - share/man/man1/cadabra2html.1
+      - share/man/man1/cadabra2python.1
+      - share/applications/cadabra2*
+    test:
+      commands:
+        - test -f $PREFIX/bin/cadabra2-gtk
+
+  - name: cadabra-jupyter-kernel
+    build:
+      skip: True  # [win]
+    requirements:
+      build:
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+      host:
+        - xeus
+        - libuuid
+        - nlohmann_json
+        - python
+        - boost-cpp
+        - jsoncpp
+        - glibmm
+      run:
+        - python
+        - boost-cpp
+        # Remove these when they get run_exports
+        - {{ pin_compatible("glibmm") }}
+        - {{ pin_subpackage("cadabra2", exact=True) }}
+      run_constrained:
+        - cadabra2-gtk {{ version }}
+
+    files:
+      - bin/cadabra-jupyter-kernel
+      - share/jupyter/kernels/cadabra/*
+    test:
+      commands:
+        - test -f $PREFIX/bin/cadabra-jupyter-kernel
+
 
 about:
   home: https://github.com/kpeeters/cadabra2
   license: GPL-3.0-only
   license_family: GPL
-  license_file: LICENSE
+  license_file:
+    - LICENSE
+    - libs/websocketpp/COPYING
+    - libs/pybind11/LICENSE
```

Comment:
@kpeeters, is this correct? Only these 3?

Reply:
I have added `libs/jsoncpp/LICENSE`, `libs/linenoise/LICENSE`, `libs/tiny-process-library/LICENSE` and `libs/whereami/LICENSE.WTFPLv2`. That should cover it.

Reply:
`libs/jsoncpp/LICENSE` is not needed because we are using the conda package here, right?

Reply:
Yes, I just added it to be complete.